service/pixelated/resources/account_recovery_resource.py

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright (c) 2017 ThoughtWorks, Inc.
+#
+# Pixelated is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Pixelated is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Pixelated. If not, see <http://www.gnu.org/licenses/>.
+
+import os
+from xml.sax import SAXParseException
+
+from pixelated.resources import BaseResource, UnAuthorizedResource
+from twisted.python.filepath import FilePath
+from pixelated.resources import get_static_folder
+from twisted.web.http import OK
+from twisted.web.resource import NoResource
+from twisted.web.template import Element, XMLFile, renderElement
+
+
+
+def parse_accept_language(all_headers):
```

Comment:
This method appears to be unused. I think we can remove it for now.

Or, if we are already implementing translation, does it make sense to extract it to `pixelated.resources` and import both on `login_resource` and `account_recovery_resource`, to avoid duplication?